src/pages/admin/admin.html

```diff
@@ -0,0 +1,6 @@
+<ion-tabs>
+  <ion-tab [root]="tab1Root" tabTitle="Notifications" tabIcon="notifications"></ion-tab>
+  <ion-tab [root]="tab2Root" tabTitle="Users" tabIcon="contacts"></ion-tab>
+  <ion-tab [root]="tab3Root" tabTitle="Billing" tabIcon="card"></ion-tab>
+  <ion-tab [root]="tab4Root" tabTitle="Test" tabIcon="pulse"></ion-tab>
```

Comment:
@tokland Did you name tabNRoot on purposes or it was a default name? I named my tab in a more specific name (actually the name of the tabs). Any of them work for me but just to understand how to proceed...